acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/Signer.java

```diff
@@ -95,6 +100,44 @@ public boolean isListening() {
     return "OK".equals(body);
   }
 
+  public HttpResponse signData(final PublicKey publicKey, final Bytes message, final Bytes domain)
```

Comment:
this should be signBlock instead of signData unless we want to pass in the endpoint as well?